### PR TITLE
Data update hierarchy

### PR DESCRIPTION
**Pull request by m-mirz**
By making a contribution to this repository, I agree to the terms indicated on [this repository](../1_Contributor-License-Agreement.md)

Follow up of PR #12 and addressing new issue.

### CODE REVIEW

File: Data-model-initial-suggestion.md [hunk 1]
```diff
@@ -32,6 +38,8 @@ Note that this may not be the best format, but I suggest we start here and evolv
 
 <br>
 
+##DYNAMIC DATA
```
Comment (gwpicard) [inline Data-model-initial-suggestion.md:41]: Is this one necessary?

Comment (Henry-WattTime) [reply Data-model-initial-suggestion.md:41]: Was intended to show the difference between regularly updated data that comes in a stream (fuel mix, price, etc) and more static data like grid topology and power plant details. Thoughts?

Comment (gwpicard) [reply Data-model-initial-suggestion.md:41]: Yes definitely agree. More that we have the header `DYNAMIC DATA` appearing twice so wondering if the second instance is redundant

Comment (Henry-WattTime) [reply Data-model-initial-suggestion.md:41]: Oops, I kept moving it around to get it in the right place.

**Review by gwpicard — APPROVED** [review]
LGTM!

**Comment by m-mirz** [comment]
I referenced this in #20. I think this can be merged on Thursday latest.

**Pull request merged**